packages/ui/src/components/BenefitCard/BenefitCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react';
+import { Box, BoxProps } from '@chakra-ui/react';
+
+import { Typography } from '../Typography';
+
+export interface BenefitCardProps extends BoxProps {
+  icon: ReactElement;
+  title?: string;
+  subtitle?: string;
+}
+
+export const BenefitCard = ({ icon, title, subtitle, ...props }: BenefitCardProps) => {
+  return (
+    <Box
```

Comment:
It would still be a little shorter if you use the `Center` component here 😉 In such a case you can remove
```
display="flex"
alignItems="center"
justifyContent="center"
```

Reply:
My bad, I only replaced it for the second Box component. Fixed now